api/socket.go

```diff
@@ -47,13 +47,17 @@ func (s *Sockets) Find(id string) *Conn {
 }
 
 func (s *Sockets) Inform(id string) {
-	s.mutex.Lock()
-	defer s.mutex.Unlock()
-
+	s.players.mutex.Lock()
 	p := s.players.Get(id)
 	if p == nil {
+		s.players.mutex.Unlock()
 		return // player doesn't exist
 	}
+	newMsgData := p.Format(id)
```

Comment:
is there a reason for this?

It's only used once - then the rewrite of newMsgData is using a different id